getting-started/pull-request-lifecycle.rst

```diff
@@ -1,5 +1,4 @@
 .. _pull-request-lifecycle:
-.. _patch:
```

Comment:
FTR in the past we left obsolete labels for backward compatibility, even if we didn't use them anymore.

Reply:
I thought the same, but I don't think this one matters too much, it's at the top of the file. Missing anchor => page loads at the top anyway.

The only problem could be if someone referenced this label from another site using Intersphinx, but I think that's unlikely here.

(But it's no big deal to keep it, either.)